qa/integration/specs/monitoring_api_spec.rb

```diff
@@ -245,6 +245,33 @@
     logging_get_assert logstash_service, "INFO", "TRACE"
   end
 
+  it "should retrieve the pipeline flow statuses" do
+    logstash_service = @fixture.get_service("logstash")
+    logstash_service.start_with_stdin
+    logstash_service.wait_for_logstash
+    number_of_events.times { logstash_service.write_to_stdin("Testing flow metrics") }
+
+    Stud.try(max_retry.times, [StandardError, RSpec::Expectations::ExpectationNotMetError]) do
+      # node_stats can fail if the stats subsystem isn't ready
+      result = logstash_service.monitoring_api.node_stats rescue nil
+      expect(result).not_to be_nil
+      # we use fetch here since we want failed fetches to raise an exception
+      # and trigger the retry block
+      flow_status = result.fetch("pipelines").fetch("main").fetch("flow")
```

Comment:
let's see on console what output we will get.
```suggestion
      flow_status = result.fetch("pipelines").fetch("main").fetch("flow")
      puts "Flow status: #{flow_status}"
```